.idea/compiler.xml

```diff
@@ -40,14 +40,35 @@
     </wildcardResourcePatterns>
     <annotationProcessing>
       <profile default="true" name="Default" enabled="true" />
+      <profile name="Gradle Imported" enabled="true">
```

Comment:
Looks like this file got regenerated again

Reply:
This one is a different file (previous was codeStyle-idea.xml), and it's well defined in .gitignore. I do not understand why git refuses to ignore it. Tried to fight it for a while, and then assumed it harmless. 
To eliminate this line, I'll have to squash all.

Reply:
Found a way to fix it while keeping the IFI and refactoring separated.